### PR TITLE
Update the permission check of the committee of meeting.clone.

### PR DESCRIPTION
**Pull request by reiterl**
Remove the permission and remove the permission test of this permission.

### CODE REVIEW

File: openslides_backend/action/actions/meeting/clone.py [hunk 1]
```diff
@@ -291,23 +291,7 @@ def build_write_request_helper(
...
-            CommitteeManagementLevel.CAN_MANAGE,
-            committee_id,
-        ):
-            raise PermissionDenied(
-                f"Missing {CommitteeManagementLevel.CAN_MANAGE.get_verbose_type()}: {CommitteeManagementLevel.CAN_MANAGE} for committee {committee_id}"
-            )
-        if (
-            payload_committee_id := instance.get("committee_id")
-        ) and payload_committee_id != committee_id:
+        if payload_committee_id := instance.get("committee_id"):
```
Comment (jsangmeister) [inline openslides_backend/action/actions/meeting/clone.py:294]: `committee_id` is not required in the payload. As stated in the linked issue, if it is not given, the target meeting is equivalent to the source meeting and the permission check has to be done as before.

File: tests/system/action/meeting/test_clone.py [hunk 2]
```diff
@@ -519,28 +519,6 @@ def test_permissions_oml_can_manage(self) -> None:
             "meeting/2", {"is_active_in_organization_id": 1, "committee_id": 2}
         )
 
-    def test_permissions_missing_meeting_committee_permission(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/meeting/test_clone.py:522]: Also leave this test in and check instead that it suceeds to "prove" that the changes work

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/meeting/clone.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/clone.py:294]: Ah, I understand.

Comment (reiterl) [reply openslides_backend/action/actions/meeting/clone.py]: If a committee_id is in the payload, I use this for the check, else I would use the committee_id from the meeting.

File: tests/system/action/meeting/test_clone.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/meeting/test_clone.py:522]: Add a test, which shows, that the permission check is in place.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**